docs/developer-guide/releasing.md

```diff
@@ -28,7 +32,7 @@ Tag UI:
 
 ```bash
 git tag $VERSION
-git push upstream $BRANCH --tags
+git push origin $BRANCH --tags
```

Comment:
I think in most cases the remote name is `upstream`, not `origin`. Can you revert this change?

Reply:
I use origin myself. I'm not precious. I'll create a var for this.